app/Jobs/ProcessPendingWebsites.php

```diff
@@ -99,14 +99,16 @@ public function handle(): void
                         $publicAdministration = $website->publicAdministration;
 
                         if ($publicAdministration->status->is(PublicAdministrationStatus::PENDING)) {
-                            $pendingUser = $publicAdministration->users()->where('status', UserStatus::PENDING)->first();
-                            if (null !== $pendingUser) {
-                                $pendingUser->publicAdministrations()->detach($publicAdministration->id);
+                            $pendingUser = $publicAdministration->users()->where('user_status', UserStatus::PENDING)->first();
```

Comment:
Qui si corregge un bug per il quale la cancellazione della PA pending dipendeva dallo stato dell'utente e non dall'associazione tra utente PA per cui, in caso di una PA attiva e una pending per lo stesso utente, la cancellazione non sarebbe avvenuta.